contracts/cw721-base/src/state.rs

```diff
@@ -123,7 +143,7 @@ where
     T: Serialize + DeserializeOwned + Clone,
 {
     // pk goes to second tuple element
-    pub owner: MultiIndex<'a, (Addr, Vec<u8>), TokenInfo<T>>,
+    pub owner: MultiIndex<'a, (Addr, Vec<u8>), TokenInfo<T>, &'a str>,
```

Comment:
So this and line 160 below, I'm significantly less confident this is the correct transformation. I need to come back to this and walk through the two query functions that depend on it.

Adding an issue to do this.